xml/System.Threading/CancellationTokenSource.xml

```diff
@@ -515,16 +515,19 @@
         <summary>Schedules a cancel operation on this <see cref="T:System.Threading.CancellationTokenSource" /> after the specified time span.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[
-            The countdown for the delay starts during this call.  When the delay expires, 
-            this <xref:System.Threading.CancellationTokenSource> is canceled, if it has
-            not been canceled already.
-            
-            Subsequent calls to `CancelAfter` will reset the delay for this <xref:System.Threading.CancellationTokenSource>, if it has not been canceled already.
+
+## Remarks
+ The countdown for the delay starts during this call. When the delay expires, this <xref:System.Threading.CancellationTokenSource> is canceled, if it has not been canceled already.
+
+ Subsequent calls to `CancelAfter` will reset the delay for this <xref:System.Threading.CancellationTokenSource>, if it has not been canceled already.  
+  
+> [!NOTE]
+>  The `TimeSpan` struct supports a larger range of time intervals than is accepted by this method. `CancelAsync` throws `System.ArgumentOutOfRangeException` if the `delay` parameter is greater than `TimeSpan.FromMilliseconds(Int32.MaxValue)` (or `TimeSpan.FromMilliseconds(UInt32.MaxValue - 1)` on some versions of .NET) up to and including `TimeSpan.MaxValue`.
```

Comment:
I had changed this to `CancelAfter`. Did you mean to put `CancelAsync`?

Reply:
Naah my bad, give me a moment, I thinking of dumping the remarks section entirely.

Reply:
Removed the note and update the constructor also.